SportHub/src/modules/SearchResultsList/components/SearchResultsList.js

```diff
@@ -86,7 +90,7 @@ const SearchResultsList = ({ contentSearchValue }) => {
 
     <div className={styles.resultsBlock}>
       <div key={"resultsCount"} className={styles.resultsCount}>
-        <h2>{contentSearchValue + " (" + articles.length + ")"}</h2>
+        <h2>{contentSearchValue + " (" + numberOfArticles + ")"}</h2>
```

Comment:
It's my personal preferences but I would rather use string interpolation like `${}`. It's more readable than a couple of + signs